lively.modules/tests/module-test.js

```diff
@@ -16,6 +16,10 @@ let S;
 describe('module loading', () => {
   beforeEach(async () => {
     S = getSystem('test', { baseURL: dir });
+    S.set('lively.transpiler', System.get('lively.transpiler'));
+    S.config({ transpiler: 'lively.transpiler' });
+    S.babelOptions = System.babelOptions;
+    S.translate = async (load) => await System.translate.bind(S)(load);
```

Comment:
Why was this block not necessary before? Has nothing to do with this concrete test module, just wondering as these lines are inserted in a lot of tests now.

Reply:
Since we migrated to the new version of SystemJS, there are a couple of things properties we further need to add to the System object for it to work fully. We can consider moving this setup code into the `getSystem()` itself, however that requires more testing and evaluating the impacts of this refactoring in the system.